### DISCUSSION TITLE
Moving cursor to specific position in keybind.

### DISCUSSION THREAD

**Discussion by aflyingpumpkin** [hyprwm/Hyprland#1130] [Q&A] [answered]
I'm trying to create a submap that emulates the workflow I am used to from X11 using keynav, to do this I need to be able to move my cursor to a position after applying some math to the cursors current position. When going through the wiki I only saw `movecursortocorner` is there a way to move the cursor to a specific position?

**Comment by vaxerski** [comment, marked as answer]
> is there a way to move the cursor to a specific position?

No
